simulator/examples/chaining.rs

```diff
@@ -2,23 +2,27 @@
 
 use embedded_graphics::fonts::Font6x8;
 use embedded_graphics::icoord;
+use embedded_graphics::pixelcolor::BinaryColor::On as C1;
```

Comment:
Can this be aliased to `On` instead? I think it's clearer than `C1` for the examples' sake.

Reply:
I didn't want to use `On` and `Off` because they have a higher chance of already being used and I wanted to show that you can easily use any name if someone doesn't like typing `BinaryColor::On` and `BinaryColor::Off` everywhere. Maybe `BcOn` and `BcOff` instead?